internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-nlb-manifest.yml

```diff
@@ -27,7 +27,7 @@ environments:
       healthcheck:
         port: 80
       additional_listeners:
-        - port: 8081/tcp
+        - port: 8081/udp
```

Comment:
can we keep both tests? for example `8080/tcp` and `8081/udp`?

Reply:
Also it would be great if we can add an e2e test making sure switching from NLB to NLBV2 is ok

Reply:
Wanxian and I talked briefly about making an e2e test. I agree, but maybe we should discuss when? I imagine it's probably high enough priority that we want an e2e test before v1.31, but shouldn't be a part of this or the next PR